Okta.Core.Automation/README.md

```diff
@@ -1,17 +1,17 @@
-##Installation
+## Installation
 
 To install the Okta PowerShell SDK:
```

Comment:
Powershell 5.0 supports the install of modules from an online gallery (https://msdn.microsoft.com/en-us/powershell/reference/5.1/powershellget/install-module?f=255&MSPPError=-2147217396). Should we use that instead of relying on a 3rd party mechanism?